internal/binutils/binutils.go

```diff
@@ -179,6 +177,26 @@ func findObjdump(paths []string) (string, bool, bool) {
 	return "", false, false
 }
 
+// findExeOnDifferentPlatforms finds and returns path to preferred binary.
+// "names" is a list of names to search on both Linux and OSX.
+// "osxNames" is a list of names specific to OSX.
+// "names" always has a higher priority than "osxNames". The order of the name
+// within each list decides its priority (e.g. the first name has a higher
+// priority than the second name in the list).
+// It returns a string with path to the binary if found, or an empty string if
+// not found; a boolean if any acceptable binary was found.
+func findExeOnDifferentPlatforms(names, osxNames []string, paths []string) (string, bool) {
```

Comment:
The name is a bit odd - we are not finding the executable on different platforms, it's more that we search it differently on different platforms.

Is findExe still used for anything now? Would it make sense to just inline findExe here (as a nested function if needed for control flow) and call this function as findExe?

Reply:
`findExe` is still used by `findObjdump` and it is a bit rough to incorporate `findObjdump` and `findExeOnDifferentPlatforms`. I was having a hard time to come up with a good function name here; maybe `findExeForDifferentPlatforms`?

Reply:
Maybe simply `chooseExe`?

Reply:
Done.